proto/monitor.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option java_multiple_files = false;
+option java_package = "io.pixelsdb.pixels.daemon";
+option java_outer_classname = "MonitorProto";
+
+package monitor.proto;
+
+
+service MonitorService {
```

Comment:
Rename MinitorService to ScalingMetricsService, and move it to turbo.proto. Do not use a separate proto file.